### PR TITLE
#245 Create table for medication event details

### PR DESCRIPTION
**Pull request by linaskitm**

### CODE REVIEW

**Review by AzisK — APPROVED** [review]
GJ

File: database/3-fake-data.sql [hunk 1]
```diff
@@ -162,3 +162,9 @@ INSERT INTO animal_cage (animal_id, cage_id) VALUES
 (2, 4),
 (3, 5),
 (4, 6);
+
+INSERT INTO medication_event_details (event_id, treatment, expenses) VALUES
+(1, 'Some pills from Prophylaxis', 20.00),
```
Comment (AzisK) [inline database/3-fake-data.sql:167]: Medication event in `events` table is only the third one. Others are not really medication events. You would need to add them in `events` table.
`(3, 'Medical', 'Medication', '2020-05-21', 'dhjbwau74a6', 'Tick medications')`

**Review by AzisK — CHANGES_REQUESTED** [review]
Medication event in `events` table is only the third one. Others are not really medication events. You would need to add them in `events` table.
`(3, 'Medical', 'Medication', '2020-05-21', 'dhjbwau74a6', 'Tick medications')`

File: database/3-fake-data.sql (hunk 1 above)
Comment (linaskitm) [reply database/3-fake-data.sql]: Sorry for late response.
Now I see why we needed these changes, we will create tables for all `*_event_details`

**Review by AzisK — APPROVED** [review]
GJ

**Pull request merged**